source/code-generation/query-converter/how-query-converter-uses-your-data.txt

```diff
@@ -0,0 +1,52 @@
+.. _rm-query-converter-data-usage:
+
+==================================
+How Query Converter Uses Your Data
+==================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Query Converter currently uses the `Azure OpenAI Service
+<https://azure.microsoft.com/en-us/products/ai-services/openai-service>`__ 
+hosted by Microsoft. 
+
```

Comment:
[Suggestion]

I know this page isn't the traditional concept page but it is still covering mostly conceptual Query Converter info, so maybe we can add a "Behavior" heading here?